src/compiler/checker.ts

```diff
@@ -46687,7 +46689,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
 
     function checkGrammarNameInLetOrConstDeclarations(name: Identifier | BindingPattern): boolean {
         if (name.kind === SyntaxKind.Identifier) {
-            if (name.originalKeywordKind === SyntaxKind.LetKeyword) {
+            if (name.escapedText === "let") {

```

Comment:
Is there a particular reason to choose to do string comparisons in some places, but `stringToToken` in others?

Reply:
Yeah - in cases where you'd just check for a single value, it doesn't make sense to do the map lookup. It's going to be faster to just check the string contents directly.